parser.go

```diff
@@ -252,7 +255,55 @@ func (p *parser) readValue(in []byte,
 		if strings.Contains(line, `\#`) {
 			line = strings.Replace(line, `\#`, "#", -1)
 		}
+	} else if lastChar == '\n' && allowPythonMultilines {
+		parserBufferPeakResult, _ := p.buf.Peek(p.buf.Buffered())
+		peakBuffer := bytes.NewBuffer(parserBufferPeakResult)
+		pythonMultiline, err := regexp.Compile("^(\\s+)([^\n]+)")
```

Comment:
Since the regexp pattern is fixed, why not make it a variable so we don't need to compile every time, e.g. `var pythonMultilinePattern := regexp.MustCompile(...)`.